Added the Support for RAC on PODMAN (#2424)
oracle/docker-images: OracleDatabase/RAC/OracleRealApplicationClusters/PODMAN_README.md

Refer Oracle Database 21c Release documentation:

> * [Real Application Clusters Installation Guide for Podman Oracle Linux x86-64](https://support.oracle.com/portal/) to make sure your environment meeting all the pre-requisites.

Reviewer: This needs to be the direct URL to the document or include instructions on how to find it or the reference must be removed.

  # touch /opt/containers/rac_host_file
  ```

> **Note:** Do not modify `/opt/containers/rac_host_file` from the host running Oracle Podman. It will be managed from within the containers.

Reviewer: ```suggestion
**Note:** Do not modify `/opt/containers/rac_host_file` from the host running Podman as it is managed by software running inside the containers.
```

### Assign networks to Oracle RAC containers

> You need to assign the Podman networks created in section 1 to containers. Execute the following commands:

Reviewer: ```suggestion
Assign the networks created in section 1 to the containers by running the following commands:
```
